python/test/test_polars.py

```diff
@@ -1,16 +1,24 @@
 import pytest
 import opendp.prelude as dp
 
+
+from .helpers import optional_dependency
+
 dp.enable_features("contrib", "honest-but-curious")
 
 
+def seed(schema):
+    pl = pytest.importorskip("polars")
+    return pl.DataFrame(None, schema, orient="row").lazy() # type: ignore[attr-defined]
+
+
 def example_series():
     pl = pytest.importorskip("polars")
     return [
-        dp.series_domain("A", dp.atom_domain(T=float)),
-        dp.series_domain("B", dp.atom_domain(T=int)),
-        dp.series_domain("C", dp.option_domain(dp.atom_domain(T=str))),
-        dp.series_domain("D", dp.atom_domain(T=int)),
+        dp.series_domain("A", dp.atom_domain(T=dp.f64)),
+        dp.series_domain("B", dp.atom_domain(T=dp.i32)),
+        dp.series_domain("C", dp.option_domain(dp.atom_domain(T=dp.String))),
+        dp.series_domain("D", dp.atom_domain(T=dp.i32)),
```

Comment:
If the python types also work, it'd be good to test them. If they don't, I'd worry about usability -- Not necessarily something to address now, but a comment pointing to an issue would be good if it's not something for right now.

Reply:
An annoying aspect is that Polars defaults to `i64`, but the default inferred type of `int` is `i32`. So if you don't specify the types exactly, then the domains will each expect `i32` but the data (a few lines below) is actually `i64`.